pkg/jq/jq.go

```diff
@@ -0,0 +1,93 @@
+// Package jq facilitates processing of JSON strings using jq expressions.
+package jq
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"math"
+	"os"
+	"strconv"
+
+	"github.com/itchyny/gojq"
+)
+
+// Evaluate a jq expression against an input and write it to an output.
+func Evaluate(input io.Reader, output io.Writer, expr string) error {
```

Comment:
Open to naming suggestions here? I also changed the argument order to have the `input` come first compared to the same function in `gh`.

Reply:
naming makes sense to me